packages/docs/docs/guides/troubleshooting.md

```diff
@@ -17,21 +17,21 @@ In this case lookup command called "Radon IDE: Diagnostics" – when executed i
 
 The extension does not currently support all types and configurations of React Native projects.
 For example, Expo Go or brownfield apps aren't supported while we are improving the compatibility of different project setups.
-Please refer to ["Who can use this"](./getting-started.md) section for more details on that.
-If your project doesn't work out of the box because of some modifications made to its setup, chances are the Radon IDE can be configured to support your modifications, please check [configuration](./launch-configuration.md) guide to learn more.
+Please refer to ["Who can use this"](/docs/getting-started/overview) section for more details on that.
+If your project doesn't work out of the box because of some modifications made to its setup, chances are the Radon IDE can be configured to support your modifications, please check [configuration](/docs/getting-started/launching) guide to learn more.
 
 ### -sec-num- Can extension locate your React Native project
 
 The extension supports monorepo-type of setups and you should be able to use it even if the app isn't in the root folder of your workspace.
 This setup however has some limitations because the extension can only work with a single application per workspace.
 If your monorepo contain multiple projects, you'll need to instruct the extension with the location of your main React Native or Expo application folder.
-This can be done using [launch configuration](./launch-configuration.md) using `appRoot` setting – please refer to [configuration guide](./launch-configuration.md) for more details.
+This can be done using [launch configuration](/docs/getting-started/launching) using `appRoot` setting – please refer to [configuration guide](/docs/getting-started/launching) for more details.
```

Comment:
Shouldn't all of them point to `guides/configuration`?

Reply:
You're absolutely right! Fixed in [2c71bd5](https://github.com/software-mansion/radon-ide/pull/689/commits/2c71bd5edec9228cd73f382fd989ddea04362fa3)